docs/iris/src/whatsnew/latest.rst

```diff
@@ -92,6 +92,11 @@ Bugs Fixed
   loading. They are now available on the :class:`~iris.coords.CellMeasure` in
   the loaded :class:`~iris.cube.Cube`. See :pull:`3800`.
 
+* the netcdf loader can now handle any grid-mapping variables with missing
+  ``false_easting`` and ``false_northing`` properties, which was previously
```

Comment:
While this started with `false_easting` and `false_northing`, the change now covers all other optionals. Is there a reason to leave those out here? Irrelevant to most people?

Reply:
I think this was the only clear actual _bug_ that resulted.

I have checked the other cases where we made a change, and I don't think they amount to anything worth talking about.  
Most are not optional parameters in CF, so they don't show the problem -- except possibly for "bad" input files.  
Some others have defaults implemented in the build_routine, or for other reasons are "masked" problems :
  * GeogCS.longitude_of_prime_meridian : not supported in CF, so not used in netcdf load
  * RotatedGeogCS.north_pole_grid_longitude : correct default provided by the build_ routine
  * TransverseMercator.scale_factor_at_central_meridian : not optional in CF
  * Stereographic.true_scale_lat : not a CF parameter (**2**)
  * LambertConformal.secant_latitudes : not optional in CF
  * Mercator.longitude_of_projection_origin : not optional in CF
  * Mercator.standard_parallel : not handled in netcdf-load : a bug! (**1**)
  * LambertAzimuthalEqualArea : latitude/longitude_of_projection_origin : not optional in CF
  * AlbersEqualArea.latitude/longitude_of_projection_origin and standard_parallels : not optional in CF

(**1**) a bug, [already mentioned](https://github.com/SciTools/iris/pull/3804#discussion_r487065304)
(**2**) actually I think there is another "bug" here : Stereographic loading is ignoring the CF "scale_factor_at_projection_origin" attribute, which should really at least be checked + warned/errored.

The above 2 errors I will create additional issues for, but they are for the future.